library/src/main/java/com/datatorrent/lib/algo/UniqueCounter.java

```diff
@@ -78,10 +78,10 @@ public void process(K tuple)
   /**
    * The output port which emits a map from keys to the number of times they occurred within an application window.
    */
-  public final transient DefaultOutputPort<HashMap<K, Integer>> count = new DefaultOutputPort<HashMap<K, Integer>>()
+  public final transient DefaultOutputPort<Map<K, Integer>> count = new DefaultOutputPort<Map<K, Integer>>()
```

Comment:
Why this change? The output port should declare the specific type. Unlike the input port, which should be least restrictive.